### ISSUE TITLE
Warn if container is not specified in container-image-name-updates

### ISSUE THREAD

**Issue by dylansmith** [CircleCI-Public/aws-ecs-orb#24]
We recently saw an issue where `aws-ecs/deploy-service-update` was not updating the image tag in our task definition. After some debugging, we found the cause was that we were not specifying the container in `container-image-name-updates`.

Our config was:

```
- aws-ecs/deploy-service-update:
          name: deploy
          family: <our-service-here>
          cluster-name: <our-cluster-here>
          container-image-name-updates: tag=${CIRCLE_SHA1}
          verify-revision-is-deployed: true
          requires:
            - build_and_publish
            - test
          filters:
            branches:
              only: master
```

Building with this configuration succeeded, but it was effectively a no-op and simply cloned the previous task definition. Since `container` is a required part of this configuration, the orb should warn and possibly fail if it is omitted.

**Comment by lokst** [comment]
Thanks for reporting this @dylansmith! We will look into it.

**Comment by Xheno** [comment]
Currently working on this, however should it simply warn or fail if the container parameter is not set?

**Comment by lokst** [comment]
@Xheno I think it'd be good to fail

**Comment by lokst** [comment]
Resolved by #39
